.github/workflows/ci.yml

```diff
@@ -1,5 +1,8 @@
 name: CI
 on: [push, pull_request]
+permissions:
```

Comment:
While the change itself is good, I think we should also change the CodeQL workflow in a similar way; see https://github.com/mozilla/pdf.js/blob/ba2b72ec00ff1a8e8cf833893479d65ae83b3505/.github/workflows/codeql.yml. We have two workflows and this way both would be in sync.